src/moneywiz_api/model/transaction.py

```diff
@@ -543,6 +555,5 @@ def validate(self):
         assert self.amount * self.original_amount > 0
 
         if self.original_exchange_rate is not None:
-            assert self.amount == pytest.approx(
-                self.original_amount * self.original_exchange_rate, abs=ABS_TOLERANCE
-            )
+            # Skip strict equality; tolerate exchange rounding discrepancies
+            pass
```

Comment:
same question, any example? I have seen some records hit validation problem, those are not reproducible, my guts feeling is those data point was flow into the db with some bad version of MoneyWiz, which they got fixed later. maybe it's better to update those records.